ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/claimantresponse/CCDResponseRejection.java

```diff
@@ -13,19 +13,25 @@
 public class CCDResponseRejection extends CCDClaimantResponse {
 
     private CCDYesNoOption freeMediationOption;
+    private String mediationPhoneNumber;
```

Comment:
why do we need these two fields in claimantresponse and under ccd defendant?

Reply:
not sure if i understand, my understanding is, defendant and claimant has different mediationPhoneNumber's, they both are different. This is how its modelled in front end and in claim store. What do you suggest we do? ping me when you are free, we will chat about this...

Reply:
@KiranVarma02 cool, All these confusion starts in CCD world where we have everything under CCDDefendant. The ClaimantResponse is also under CCDDefendant, so was wondering if its same. My bad.